src/main/kotlin/io/reactivex/rxkotlin/Flowables.kt

```diff
@@ -0,0 +1,147 @@
+package io.reactivex.rxkotlin
+
+import io.reactivex.Flowable
+import io.reactivex.functions.BiFunction
+import io.reactivex.functions.Function3
+import io.reactivex.functions.Function4
+import io.reactivex.functions.Function5
+import org.reactivestreams.Publisher
+
+
+object Flowables {
+
+    inline fun <T1,T2,R> combineLatest(source1: Flowable<T1>, source2: Flowable<T2>, crossinline combineFunction: (T1, T2) -> R) =
```

Comment:
nit: I know people aren't often going to be reading this file, but I feel `combiner` is a nicer name than `combineFunction`.